src/map/utils/zoneutils.cpp

```diff
@@ -755,11 +755,27 @@ namespace zoneutils
             g_PZoneList[0] = CreateZone(0);
         }
 
+        // clang-format off
+        std::for_each(
+#ifndef __APPLE__ // TODO: OSX machines in CI don't have a compiler compliant with this
+            std::execution::par,
```

Comment:
I run locally on mac os. What shall I do about testing this / building?

Reply:
> OSX machines in CI

You don't have to do anything